apps/docs/content/troubleshooting/connection-closed-state-authentication

```diff
@@ -0,0 +1,10 @@
+---
+title = "Connection closed when state was authentication"
+topics = ["database", "supavisor"]
+keywords = ["connections", "pooler", "authentication"]
+
+[[errors]]
+message = "Connection closed when state was authentication"
+---
+
+This error happens when either the database doesn’t exist or if the user doesn't have the right credentials.
```

Comment:
Is there any suggested resolution? What should the user do?

Reply:
check the username and database name, and ensure that the project reference is included in the username